gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/watermarker/MultiKeyValueLongWatermark.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.data.management.conversion.hive.watermarker;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import lombok.Getter;
+
+import com.google.common.collect.Maps;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+
+import gobblin.source.extractor.Watermark;
+
+/**
+ * A {@link Watermark} that holds multiple key value watermarks. The key is a {@link String} and value is a {@link Long}.
```

Comment:
> The key is a {@link String} and value is a {@link Long}.

I think this is pretty obvious from the interface. I'd remove this comment as it has little informational content. Are there any restrictions/expectations about the keys and/or values that need to be called out? What is the semantics of the key? E.g. are the watermarks expected to be non-negative? Can they be null?